migrations/2018-08-28-010336_init/up.sql

```diff
@@ -0,0 +1,70 @@
+CREATE DATABASE IF NOT EXISTS `syncstorage` /*!40100 DEFAULT CHARACTER SET latin1 */;
+
+USE `syncstorage`;
+
+-- DROP TABLE IF EXISTS `bso`;
+CREATE TABLE `bso` (
+    `user_id` INT                           NOT NULL,
+    `collection_id` INT                     NOT NULL,
+    `id` VARCHAR(64)                        NOT NULL,
+
+    `sortindex` INT DEFAULT NULL,
+
+    `payload` MEDIUMTEXT                    NOT NULL,
+    `payload_size` INT DEFAULT '0'          NOT NULL,
```

Comment:
This `payload_size` is a separate column to (in theory) make it quicker to calculate total size of items stored in a collection.  IIRC we don't ever do that in practice, so it may be worth considering whether that optimization still makes sense for us here.

Reply:
payload_size does appear to be utilized in the info/quota/collectionion_usage calls

Reply:
It might be interesting to compare `SELECT SUM(payload_size)` vs `SELECT SUM(LENGTH(payload))` for this purpose to see whether having it as a separate column really provides much value in practice.

To be clear, I don't have any particular objection to keeping it, just wondering whether the extra complexity pays for itself or not.